### DISCUSSION TITLE
Can you share views or frontend form and listing

### DISCUSSION THREAD

**Discussion by umefarooq** [coderflexx/laravel-ticket#32] [Q&A]
Hi, this package look good for small project instead of installing separate ticketing system. Can you share view or frontend screens how are the forms and listing of tickets, categories other features in this package.

I will be really thankful for help and support.

**Comment by ousid** [comment]
Hi, @umefarooq 

I am really not following here, however you can use this package as you like, in the end. It's just a model with controller and a migration.

Let me know if you still need help.

**Comment by umefarooq** [comment]
Thank you for your answer i just want to see how other developer using this package, other question can i install this with laravel 8. i have check your composer it support php8.1

**Reply by ousid** [reply]
Unfortunately, you cannot install it with Laravel v8, it supports v9 and v10. V8 is no longer supported anyway.

**Reply by umefarooq** [reply]
Thank you.
